packages/angular_devkit/architect/src/workspace-schema.json

```diff
@@ -0,0 +1,106 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "id": "BuildFacaceWorkspaceSchema",
+  "title": "Workspace schema for validating a Architect workspace configuration file.",
+  "type": "object",
+  "properties": {
+    "$schema": {
+      "type": "string",
+      "description": "Link to schema."
+    },
+    "name": {
+      "type": "string",
+      "description": "Workspace name."
+    },
+    "version": {
+      "type": "number",
+      "description": "Workspace Schema version."
+    },
+    "root": {
+      "type": "string",
+      "description": "Workspace root."
+    },
+    "defaultProject": {
+      "type": "string",
+      "description": "Default target to run."
+    },
+    "projects": {
```

Comment:
I would like that but it might be a moot point with working schema `$refs`. As is it's slightly out of scope of the initial implementation, but definitely an important enhancement for the future.

Reply:
Out of scope for now. My `pre`/`post` transform PR made it trivial to implement, so I'm fine with this PR as is.